### ISSUE TITLE
Taking your app from Beta testing to Publishing

### ISSUE THREAD

**Issue by zassmin** [codepath/android_guides#121]
Things I see this having: 

* Sharing the different options for making it easy for an android user to test your app before uploading to the play store
* Publishing to the play store check list
  * for example, know how to create a signed apk and why that matters

**Comment by zassmin** [comment]
The checklist piece seems to be covered here, https://github.com/codepath/android_guides/issues/69, getting through the different errors is painful, for example, something as simple as not properly naming your package is something I wish I knew initially. 

Error I'm referring to: 
<img width="614" alt="screen shot 2015-12-07 at 10 08 14 pm" src="https://cloud.githubusercontent.com/assets/1654151/11672128/84df460e-9dc2-11e5-9cbe-d3675b85dcab.png">

**Comment by nesquena** [comment]
Yep, I agree we'll need to add several guides [focused on production](https://github.com/codepath/android_guides/labels/production). In particular, we should [expand this publishing guide](http://guides.codepath.com/android/Publishing-to-the-Play-Store) and address this issue along with #69. I've also now documented this category in the [master topic needed list](https://github.com/codepath/android_guides/issues/2).

**Comment by nair25** [comment]
Try Flybuilds - Flybuilds is the fastest way for developers to create a team and share in-development iOS/Android App for beta testing through their personal cloud in a highly secure manner.
https://www.flybuilds.com/
